ironfish/src/sdk.test.ts

```diff
@@ -131,7 +131,7 @@ describe('IronfishSdk', () => {
 
         expect(openDb).toHaveBeenCalledTimes(1)
         expect(client).toBeInstanceOf(RpcMemoryClient)
-        expect((client as RpcMemoryClient)._node).toBe(node)
+        expect((client as RpcMemoryClient).router.server.context.node).toBe(node)
```

Comment:
We should delete this since this field is gone. It's just a router expect / test now.

Reply:
It doesn't have anything to do with memory client but it is related to SDK which I think is what this is testing. It's testing that the SDK does actually attach the correct node